src/Sulu/Bundle/ContactBundle/Entity/ContactRepository.php

```diff
@@ -255,7 +255,8 @@ public function findGetAll($limit = null, $offset = null, $sorting = null, $wher
             ->leftJoin('u.faxes', 'faxes')
             ->leftJoin('u.contactAddresses', 'contactAddresses')
             ->leftJoin('contactAddresses.address', 'addresses')
-            ->leftJoin('u.account', 'account')
```

Comment:
@danrot: Yes, the line lead to the following error:
`[Semantical Error] line 0, col 303 near 'account LEFT': Error: Class Sulu\Bundle\ContactBundle\Entity\Contact has no association named account`